### ISSUE TITLE
The new in-add-on build dependencies process fails if the setting "Save configuration when exiting NVDA" is set to off

### ISSUE THREAD

**Issue by rperez030** [aaclause/nvda-OpenAI#43]
This probably doesn't require much clarification. If the setting to save the configuration is off, the update section doesn't get saved in the .ini file before restarting NVDA, and the process enters a loop. It may be a good idea to keep the setting to search for updates as part of the configuration but save the update state in an add-on file.
